rules/storyboarding-do-you-conduct-specification-analysis-by-creating-mock-ups/rule.md

```diff
@@ -38,14 +40,12 @@ Start by identifying the key features and functionality that product should have
 
 Then create a basic wireframe of your design. A wireframe is a visual representation of the layout and structure of your website or app. You can use software tools like Figma , Adobe XD, or Sketch to create digital wireframes. Alternatively, you can use paper and pencil to sketch out a rough wireframe by hand. Add details to your wireframe by adding boxes and labels for different features and elements (e.g. buttons, input fields, and images). You can also annotate a wireframe or storyboard with brief descriptions of intended interactions and elements.
 
-![Figure: Example of a hand-drawn mockup. Nice and quick for early concept design](Hand-Drawn-Mockup.jpg)
-
-![Figure: Example of wireframes. Created in Figma](figma_wireframe_app_screenshot.png)
+![Figure: Example of a hand-drawn mockup. Nice and quick for early concept design](hand-drawn-mockup.jpg)
 
+![](figma_wireframe_app_screenshot.png)
 ::: good
-![Figure: This wireframe indicates priorities of features](wireframe-with-notes.jpg)
+Figure: Good Example – Example of wireframes. Created in Figma
```

Comment:
@Geordie88 

see the right markdown for images examples and fix all in this rule: https://ssw.com.au/rules/rule/#6-images